### DISCUSSION TITLE
Can we set single date selection for month view instead of date range?

### DISCUSSION THREAD

**Discussion by abhuzz** [kvyatkovskys/KVKCalendar#320] [Q&A]
![single selection date](https://github.com/kvyatkovskys/KVKCalendar/assets/6681394/39c80500-f04a-456b-99ba-d8abb51ea575)
 How can I set a single selection on month view style instead of date range?
can we have any property to set range selection should be off and on?

**Comment by kvyatkovskys** [comment]
Set selection mode `style.month.selectionMode = .single`
